pkg/server/router/credential_test.go

```diff
@@ -261,6 +261,11 @@ func TestCredentialRouter(t *testing.T) {
 				assert.Empty(tt, createdCred)
 				assert.Error(tt, err)
 				assert.ErrorContains(tt, err, "cannot use revoked key")
+
+				// create a schema with the revoked key, it fails
+				_, err = schemaService.CreateSchema(context.Background(), schema.CreateSchemaRequest{Issuer: controllerDID.DID.ID, Name: "schema (revoked key)", Schema: getEmailSchema(), FullyQualifiedVerificationMethodID: keyID})
```

Comment:
Seems like the wrong file to test this in. Can you move to `router/schema_test.go`?

Reply:
Yes, I can. I put the test in this file because it seemed like a simplest way to test the change with minimal number of new lines (since the key revocation logic is already in place), and because schema creation was [already being tested in this file](https://github.com/TBD54566975/ssi-service/pull/579/files/73c769648a98d0d194c3c445618cc57a2410c7f6#diff-c16fc760aec028bd13fafd51b015b31db88fbd79a1e130f3097fc3010b838ac7R149) anyways.